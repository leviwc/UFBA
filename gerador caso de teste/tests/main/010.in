30 230
15 1
11 17
16 12
15 20
23 10
6 2
26 21
4 18
27 14
26 28
27 5
22 4
21 28
3 3
12 21
24 26
12 24
1 6
22 15
27 26
20 12
2 29
26 6
23 20
28 13
6 24
15 14
6 11
17 14
23 28
24 10
4 24
23 13
18 13
10 29
2 30
5 22
10 26
8 8
2 26
4 29
26 13
16 4
28 12
4 4
10 14
25 22
11 3
2 11
6 16
13 1
12 4
7 2
21 11
1 17
25 9
23 22
15 9
29 13
27 2
14 18
2 19
3 9
22 10
28 9
15 13
19 12
8 2
29 8
9 21
18 16
8 4
18 27
17 20
25 27
5 19
1 19
16 23
5 10
21 10
25 13
1 7
28 17
11 4
19 15
17 19
4 2
14 11
7 28
13 5
8 12
26 5
13 19
21 17
25 21
19 20
22 19
19 7
18 21
11 8
23 25
6 21
1 10
3 8
13 7
20 2
21 29
3 27
21 16
27 12
21 24
15 3
6 12
16 11
25 3
3 20
22 17
22 28
22 1
18 7
9 12
8 26
1 8
2 17
2 5
15 16
4 7
13 9
7 23
23 30
26 15
9 14
8 5
22 20
7 10
23 14
3 14
30 14
29 3
5 25
30 18
19 18
25 25
24 18
8 19
29 7
8 17
4 3
6 14
17 15
24 27
5 6
19 30
5 11
10 3
5 16
9 29
9 24
11 15
29 14
1 30
10 2
16 10
13 24
24 5
26 18
12 3
19 4
7 12
15 12
6 10
28 24
29 16
23 11
9 8
17 10
5 12
26 3
21 5
5 23
11 19
21 21
13 3
2 9
19 10
20 29
26 29
23 29
23 19
11 22
26 7
7 24
26 23
12 13
29 27
3 6
20 26
9 4
20 5
1 25
23 18
23 3
16 3
17 5
6 6
14 19
1 29
16 2
17 24
20 9
20 20
15 27
17 4
16 9
29 5
3 30
15 29
13 2
15 15
3 21
22 14
20 21
4 23
13 20
28 1
28 6
29 24
30 30
18 6
22 7
