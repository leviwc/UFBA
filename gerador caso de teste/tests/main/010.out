Resultado com busca binaria: 6
7192.562000 s
Resultado com backtracking apenas: 6
0.249000 s
Resultado testando todas as possibilidades: 6
44098.060000 s
Resultado com busca binaria otimizada: 6
1398.724000 s
